### DISCUSSION TITLE
IS there a way to diagnose why body-footer is going in above the blog list?

### DISCUSSION THREAD

**Discussion by ggirton** [quarto-dev/quarto-cli#939] [Q&A] [answered]
I have made the _quarto yaml that looks like this, for a blog

project:
  type: website

website:
  title: "mvpblog"
  navbar:
    right:
      - about.qmd
      - icon: github
        href: https://github.com/
      - icon: twitter
        href: https://twitter.com
  body-footer: |
        ---   all photos Copyright &copy; 2022 MeMeMe all rights reserved

format:
  html:
    theme: cosmo
    css: styles.css

Then, the stock index.qmd, like this:

---
title: "mvpblog"
listing:
  contents: posts
  sort: "date desc"
  type: default
  categories: true
  sort-ui: false
  filter-ui: false
page-layout: full
title-block-banner: true
---

The result is that the body footer comes out ABOVE the blog entries in the listing (not after them, as I sort of expected).
I know there must be a way for me to set up logging so I can see where in the process my specification of the body footer is going awry.  It's even possible that I'm doing everythnig right and there is a bug!  (note from me: this has pretty much never happened before, though).

What I lack is an understanding of how to turn on dedbugging instrumentation, so I can get the dedbugs ded.  I mean, debugging, but you know what I mean.  Does it think it is doing what I think it should be doing, etc. etct.

Thank you, TIA!

**Comment by ggirton** [comment]
I was able to make the page render the way I wanted by moving the "body-footer" line above the nav-bar, where I was able to see that it was indented incorrectly.  There was still a spurious error message about something in line 3, but that went away as I reformatted.  The desired footer appeared.

The quarto documentation of body-footer in the yaml was helpful in diagnosing this, but I still wish I could see the version of Quarto I am using.  Meanwhile, come visit me anytime at minimum viable python dot com

**Comment by jjallaire** [comment, marked as answer]
you can do `quarto --version` to the version. A meta `generator` tag is also written into the HTML which indicates the version used to build the page.

**Reply by ggirton** [reply]
Thank you, JJ.  I see <meta name="generator" content="quarto-1.0.38"> in the html, and quarto --version now works (for the 1st time, I guess after the update.  The prior version must have been inchoate to RStudio.  I do NOT require to dig out this meta generator, but since I saw someone else asked a question about versioning (and thought it might be fun) I tried various variations of the shortcode {{< meta generator >}}   Alas ... to no avail.  The tag must be upstream at that point, or a different identifier used to get at it.  All best && cheers George
